trajopt/include/trajopt/problem_description.hpp

```diff
@@ -358,7 +360,8 @@ struct JointVelTermInfo : public TermInfo
   void hatch(TrajOptProb& prob) override;
   DEFINE_CREATE(JointVelTermInfo)
 
-  JointVelTermInfo() : TermInfo(TT_COST | TT_CNT) {}
+  /** @brief Initialize term with it's supported types */
+  JointVelTermInfo() : TermInfo(TT_COST | TT_CNT | TT_USE_TIME) {}
```

Comment:
should it be this way? Issue it is not possible to just have the term type as TT_USE_TIME.
```suggestion
   JointVelTermInfo() : TermInfo(TT_COST | TT_CNT | (TT_COST + TT_USE_TIME)| ( TT_CNT + TT_USE_TIME)) {}
```

Reply:
These should be equivalent. 001 | 010 | 100 = 111 and 001 | 010 | 101 | 110 = 111. 

Now another issue that does occur to me is that it is currently a bit ambiguous whether a cost that does not have time is set that way or just hasn't been set yet.